pkg/lib/config/authentication_flow_captcha.go

```diff
@@ -4,12 +4,60 @@ var _ = Schema.Add("AuthenticationFlowCaptcha", `
 {
 	"type": "object",
 	"additionalProperties": false,
+	"required": ["mode"],
 	"properties": {
-		"required": { "type": "boolean" }
+		"mode": { "type": "string", "enum": ["never", "always"] },
+		"fail_open": { "type": "boolean" }
```

Comment:
No need to add this now. Generally, do not add something that is not going to be implemented now